pkg/network/ndmz/ndmz.go

```diff
@@ -104,7 +107,7 @@ func Create() error {
 			return err
 		}
 		if !received {
-			return errors.Errorf("public interface in ndmz didn't not received an IP. make sure dhcp is working")
+			return errors.Errorf("public interface in ndmz didn't received an IP. make sure dhcp is working")
```

Comment:
I prefer to not use abbreviation in error message.
```suggestion
			return errors.Errorf("public interface in ndmz did not received an IP. make sure dhcp is working")
```